web/src/routes/PluginMain.tsx

```diff
@@ -67,14 +68,22 @@ function PluginMain() {
   }
 
   const quayConfig = useQuayConfig();
-  const {user, loading, error} = useCurrentUser();
-
   const setIsPluginState = useSetRecoilState(IsPluginState);
   const [isConfirmUserModalOpen, setConfirmUserModalOpen] = useState(false);
+  const [tokenReady, setTokenReady] = useState(false);
 
-  chrome?.auth?.getToken().then((token) => {
-    GlobalAuthState.bearerToken = token;
-  });
+  useEffect(() => {
+    if (chrome?.auth) {
+      chrome.auth.getToken().then((token) => {
+        GlobalAuthState.bearerToken = token;
+        setTokenReady(true);
+      });
+    } else {
+      setTokenReady(true);
+    }
+  }, [chrome]);
```

Comment:
IIRC the input here has to be a primitive (number, string) otherwise if it's an object it does incorrect renders